src/dialog-editor/services/dialogValidationService.ts

```diff
@@ -31,14 +31,21 @@ export default class DialogValidationService {
         field => ({ status: ! _.isEmpty(field.label),
                     errorMessage: __('Dialog field needs to have a label') }),
         field => ({ status: ! ((field.type === 'DialogFieldDropDownList' ||
-                              field.type === 'DialogFieldRadioButton')
-                             && (!field.dynamic && _.isEmpty(field.values))),
+                                field.type === 'DialogFieldRadioButton')
+                               && (!field.dynamic && _.isEmpty(field.values))),
                     errorMessage: __('Dropdown needs to have entries') }),
         field => ({ status: ! (field.type === 'DialogFieldTagControl'
                                && field.category_id === ''),
                     errorMessage: __('Category needs to be set for TagControl field') }),
         field => ({ status: ! (field.dynamic && _.isEmpty(field.resource_action.ae_class)),
                     errorMessage: __('Entry Point needs to be set for Dynamic elements') }),
+        field => ({ status: ! ((field.type === 'DialogFieldDropDownList' ||
+                                field.type === 'DialogFieldRadioButton')
+                               && (field.data_type === 'integer')
+                               && (!_.chain(field.values)
+                                   .map(dialog_entries => _.toNumber(dialog_entries[0]))
+                                   .every(value => !_.isNaN(value)).value())),
+                    errorMessage: __('Value type is set as Integer, but entries values are not numbers')}),
```

Comment:
The text does not look correct. Can some native speaker review? @h-kataria, @eclarizio ?

Reply:
Yeah, "entries values" is a bit confusing. Here's my suggestion, though it could probably be improved. I'm not sure if we have any sort of standard wording for these kinds of things.

`Value type is set as Integer, but the value entered is not a number` or
`Value type is set as Integer, but the value entered is not an integer`.